tests/test_odb.py

```diff
@@ -0,0 +1,15 @@
+from io import BytesIO
+
+import pytest
+
+from dvc_objects.db import ObjectDB
+
+
+@pytest.mark.parametrize(
+    "data, expected",
+    [(b"content", b"content"), (BytesIO(b"content"), b"content")],
+)
+def test_write(memfs, data, expected):
+    odb = ObjectDB(memfs)
+    odb.write("1234", data)
+    assert memfs.cat_file("/12/34") == expected
```

Comment:
Do we need odb.read too?
```suggestion
    assert odb.read(oid) == expected
```
```suggestion
    assert memfs.cat_file("/12/34") == expected
```